src/site/stages/build/index.js

```diff
@@ -159,6 +160,7 @@ function defaultBuild(BUILD_OPTIONS) {
   smith.use(createBuildSettings(BUILD_OPTIONS));
 
   smith.use(updateExternalLinks(BUILD_OPTIONS));
+  smith.use(addSubheadingsIds(BUILD_OPTIONS));
```

Comment:
We should also add this to the `stages/preview` pipeline.

Reply:
added